argmin-py/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "argmin-py"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+name = "argmin"
+crate-type = ["cdylib"]
+
+[dependencies]
+anyhow = "1.0.70"
+argmin_testfunctions = "0.1.1"
+argmin = {path="../argmin", default-features=false, features=[]}
+argmin-math = {path="../argmin-math", features=["ndarray_latest-serde"]}
+ndarray-linalg = { version = "0.16", features = ["netlib"] }
+ndarray = { version = "0.15", features = ["serde-1"] }
```

Comment:
This section requires cleanup, I am not sure what's the best configuration of features.

Reply:
This looks mostly fine. I think in the long run at least the `serde1` feature of argmin can be enabled because that would allow checkpointing. But I guess checkpointing will need more work anyways.
At some point we will have to decide which BLAS backend to use. This is probably mostly a platform issue (only Intel-MKL works on Linux, Windows and Mac) and a licensing issue since the compiled code will be packaged into a python module.